src/records/Records.jsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 import { GridRecords, GridStyles } from '../components/Grid';
 import { PrimaryText } from '../components/Text';
+import { useRecords } from '../context/recordsContext';
 
-export const Records = () => (
-  <GridStyles>
-    <PrimaryText weight='bold'>Dashboard</PrimaryText>
-    <PrimaryText>Here you can find all your records.</PrimaryText>
-    <GridRecords>
-      <div>1</div>
-      <div>2</div>
-      <div>3</div>
-      <div>4</div>
-    </GridRecords>
-  </GridStyles>
-);
+export const Records = () => {
+  const { records } = useRecords();
+  return (
+    <GridStyles>
+      <PrimaryText weight='bold'>Dashboard</PrimaryText>
+      <PrimaryText>Here you can find all your records.</PrimaryText>
+      <GridRecords>
+        {records.map((record) => (
+          <div key={record.title}>{record.title}</div>
```

Comment:
think its just for testing but here just need to be the cover.

Reply:
I wanted to create a little box with the cover and the title. I have the cover but there is no image and it look weird.

Reply:
yes, teh covers are not final yet

Reply:
I saw you created a shakira record :D But the cover is coming from the local. I remember Vasilis couldn't solve the path thing, do you know how to solve it?